atlasdb-config/src/main/java/com/palantir/atlasdb/config/TargetedSweepRuntimeConfig.java

```diff
@@ -42,7 +42,7 @@ public boolean enabled() {
      */
     @Value.Default
     public int shards() {
-        return AtlasDbConstants.DEFAULT_SWEEP_QUEUE_SHARDS;
```

Comment:
does this constant have any other references? should we delete it from that class?

Reply:
It is used in tests and, more importantly, as the default value when reading the persisted number of shards and no value is present -- we take the min of this and the runtime config.

Reply:
maybe leave a comment as to why it's not the default? Logic makes sense, but it's still a bit surprising